pcsx2/GS/Renderers/HW/GSRendererHW.cpp

```diff
@@ -5374,22 +5365,17 @@ __ri void GSRendererHW::DrawPrims(GSTextureCache::Target* rt, GSTextureCache::Ta
 			if (rt_new_alpha_max == rt_new_alpha_min)
 				rt->m_alpha_range = false;
 		}
+
+		if (DATE)
+		{
+			blend_alpha_min = std::min(blend_alpha_min, 127 + static_cast<int>(m_cached_ctx.TEST.DATM));
+			blend_alpha_max = std::min(blend_alpha_max, 127 + static_cast<int>(m_cached_ctx.TEST.DATM));
```

Comment:
This is not the same as the code you removed.

say blend_alpha_min is 160 and DATM is 1, you just made the min 128, instead of 160 (which it should be.

Reply:
Oops, you're right, didn't notice that.